src/cbor_builders.cc

```diff
@@ -286,20 +298,29 @@ void AuthenticatorClientPinCborBuilder::SetPinHashEnc(
   SetMapEntry(ClientPinParameters::kPinHashEnc, cbor::Value(pin_hash_enc));
 }
 
+void AuthenticatorClientPinCborBuilder::SetDefaultPermissions() {
+  SetMapEntry(ClientPinParameters::kPermissions, cbor::Value(0x03));
+}
+
+void AuthenticatorClientPinCborBuilder::SetPermissionsRpId(
+    const std::string& rp_id) {
```

Comment:
If `cbor::Value(rp_id)` below creates a `std::string`, it's better to take `rp_id` by value and move it below. Otherwise, it should hopefully work to take `std::string_view`. But usually taking `const std::string&` as argument should be rare.

Reply:
I changed the calls to move semantics. The third_party code in its current, ported form doesn't take `string_view`.